api/registrations/serializers.py

```diff
@@ -773,11 +774,8 @@ def _is_attached_file_valid(self, file_metadata, registering):
             return False
 
         # Confirm that the file has the expected name
-        normalized_file_names = [
-            normalize('NFD', file_metadata['file_name']),
-            normalize('NFC', file_metadata['file_name']),
-        ]
-        if attached_file.name not in normalized_file_names:
+        normalized_file_name = normalize_unicode_filename(file_metadata['file_name'])
+        if attached_file.name not in normalized_file_name:
```

Comment:
If this doesn't prevent unicode characters from being in the file name during the registration, what is happening here?

Reply:
The intention is to ensure that the file name matches expected unicode-normalized forms. Is there a better approach or am I missing something?